definitions/tools/check_strand.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "how_are_we_stranded_here"
+baseCommand: ['check_strandedness']
+arguments: [
+    "--print_commands"
+]
+
+stdout: "$(inputs.reads1.nameroot).strandness_check.txt"
+
+requirements:
+    - class: ResourceRequirement
+      ramMin: 16000
+      tmpdirMin: 25000
+      coresMin: 4
+    - class: DockerRequirement
+      dockerPull: "smk5g5/checkstranded:latest"
+inputs:
+    gtf_file:
+        type: File
+        inputBinding:
+            prefix: --gtf
+    kallisto_index:
+        type: File
+        inputBinding:
+            prefix: --kallisto_index
+    cdna_fasta:
+        type: File
+        inputBinding:
+            prefix: --transcripts
+    reads1:
```

Comment:
maybe these could be renamed "fastq_reads1" to make it very clear what's expected?  (a doc line could help too!)